README.md

```diff
@@ -38,15 +39,40 @@ python3 -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html
 
 ## Currently supported
 
-Operations - Contraction Tensor
-Data Types - FP32 , FP64
+### Operation: Contraction tensor
+
+hiptensor supports a tensor contraction of the form $D = \alpha AB + \beta C$
+
+Supported data-type combinations are:
+
+| typeA       | typeB       | typeC       | typeCompute       | notes                              |
+|-------------|-------------|-------------|-------------------|------------------------------------|
+| bf16        | bf16        | bf16        | f32               |                                    |
```

Comment:
Should the letters in this table not be in caps? (e.g., BF16)

Reply:
I think we can keep the bf16 to be consistent with the RocWMMA's format. 

RocWMMA's readme

![image](https://github.com/ROCm/hipTensor/assets/142121551/ca664024-8689-4845-8c6a-96e993470fd5)